src/Plugins/Markdown/MdRenderer.cs

```diff
@@ -95,20 +99,41 @@ private MdRenderer Property(DocTypeDeclaration parent, DocProperty property)
         .Returns(indexer.Comment);
 
     // TODO: We can omit rendering `Name` and render `Declaration` only but it'd be nice to make this customizable via plugins.
-    private MdRenderer Name(DocMember member) => member switch
+    private MdRenderer Name(DocMember member)
     {
-        DocMethod method =>
-            Line($"{new string(c: '#', _level)} " +
-                 $"{method.Name.Surround("~~", "~~", when: method.Deprecated)}" +
-                 $"{method.TypeParams.Select(x => x.Name).Separated(", ").Surround("<", ">")}" +
-                 $"({method.Params.Select(x => x.Type?.FullName).NonNulls().Separated(", ")})"),
-        DocIndexer indexer =>
-            Line($"{new string(c: '#', _level)} this[{indexer.Params.Select(x => x.Type?.Name).NonNulls().Separated(", ")}]"),
-        DocTypeDeclaration type when _level is 1 =>
-            Line($"{new string(c: '#', _level)} {type.FullyQualifiedName.Surround("~~", "~~", when: type.Deprecated)}{type.TypeParams.Select(x => x.Name).Separated(with: ", ").Surround("<", ">")}"),
-        _ =>
-            Line($"{new string(c: '#', _level)} {member.Name.Surround("~~", "~~", when: member.Deprecated)}"),
-    };
+        return member switch
+        {
+            DocMethod method =>
+                Line($"{new string(c: '#', _level)} " +
+                     Link(
+                         $"{method.Name.Surround("~~", "~~", when: method.Deprecated)}" +
+                         $"{method.TypeParams.Select(x => x.Name).Separated(", ").Surround("<", ">")}" +
+                         $"({method.Params.Select(x => x.Type?.FullName).NonNulls().Separated(", ")})")),
+            DocIndexer indexer =>
+                Line($"{new string(c: '#', _level)} " +
+                     Link($"this[{indexer.Params.Select(x => x.Type?.Name).NonNulls().Separated(", ")}]")),
+            DocTypeDeclaration type when _level is 1 =>
+                Line($"{new string(c: '#', _level)} " +
+                     Link($"{type.FullyQualifiedName.Surround("~~", "~~", when: type.Deprecated)}" +
+                          $"{type.TypeParams.Select(x => x.Name).Separated(with: ", ").Surround("<", ">")}")),
+            _ =>
+                Line($"{new string(c: '#', _level)} " +
+                     Link($"{member.Name.Surround("~~", "~~", when: member.Deprecated)}")),
+        };
+
+        string Link(string text)
+        {
+            if (member.Location is not null)
+            {
+                var from = new Uri(_output);
+                var to = new Uri(member.Location.Path);
+
+                return $"[{text}]({from.MakeRelativeUri(to)}#L{member.Location.Start.Line + 1})";
```

Comment:
The line number format should be configurable. For example, GitHub uses `#L3-L7`, while Bitbucket uses `#lines-3:7`.

Reply:
What do you think about extracting an `MdRendererOptions` for tweaking such things?

For example, we could call the `UseMdRenderer` method on the pipeline with few ways:
```cs
pipeline
  .UseMdRenderer("...");
```
or
```cs
pipeline
  .UseMdRenderer(options => options
    .CleanupOutputDir()
    .UseGitHubMarkdown())
```

Reply:
I think it's a really good idea! I was also planning to raise a ticket to create a configuration mechanism for the pipeline, which I'll do shortly. 👍